scripts/run-integration-tests.sh

```diff
@@ -201,7 +203,7 @@ if [[ $TEST_PASS -eq 0 && "$RUN_CONFORMANCE" == true ]]; then
 
   go install github.com/onsi/ginkgo/ginkgo
   wget -qO- https://dl.k8s.io/v$K8S_VERSION/kubernetes-test.tar.gz | tar -zxvf - --strip-components=4 -C /tmp  kubernetes/platforms/linux/amd64/e2e.test
-  ginkgo -p --focus="Conformance"  --failFast --flakeAttempts 2 \
+  $GOPATH/bin/ginkgo -p --focus="Conformance"  --failFast --flakeAttempts 2 \
```

Comment:
This should not really have been necessary, since `go install` on line 202 above should have already installed the ginkgo binary into the script's PATH. What is this specifically fixing? Since the git commit message/summary doesn't provide any details as to a bug/error, it's tough to tell.

Reply:
Hi @jaypipes,

While executing this script through codepipeline, ginkgo was failing with the below error so had to add the full path. Other option was to set $PATH. 

This is mentioned in ginkgo installation readme - "This fetches ginkgo and installs the ginkgo executable under $GOPATH/bin – you'll want that on your $PATH"

Running conformance tests against cluster.
go: downloading github.com/onsi/ginkgo v1.11.0
go: extracting github.com/onsi/ginkgo v1.11.0
go: downloading golang.org/x/sys v0.0.0-20191026070338-33540a1f6037
go: extracting golang.org/x/sys v0.0.0-20191026070338-33540a1f6037
go: finding github.com/onsi/ginkgo v1.11.0
go: finding golang.org/x/sys v0.0.0-20191026070338-33540a1f6037
kubernetes/platforms/linux/amd64/e2e.test
**./scripts/run-integration-tests.sh: line 205: ginkgo: command not found**
Cluster was provisioned already. Deprovisioning it...